.github/workflows/upload_env_image.yml

```diff
@@ -0,0 +1,99 @@
+name: Upload Env Images
+on:
+  workflow_dispatch: {}
+  push:
+    paths:
+      - 'images/build-env/Dockerfile'
+      - 'images/dev-env/Dockerfile'
+    branches:
+      - master
+      - release-*
+
+jobs:
+  build-specific-architecture:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+        image: [dev, build]
+    outputs:
+      image_tag: ${{ steps.image_tag.outputs.image_tag }}
+    steps:
+      - uses: actions/checkout@master
+        with:
+          # Must use at least depth 2!
```

Comment:
why?

Reply:
I don't know. It's copied from other yaml in this probject. Maybe it will affect the behavior of `GITHUB_REF`.

I will remove this line and try whether it works well.

Reply:
Fixed.